### ISSUE TITLE
file-like object mp3 doesn't work but file.mp3 works

### ISSUE THREAD

**Issue by JoergZ2** [pygame/pygame#2407]
System: Ubuntu 20.04
Python: 3.8
pygame: just installed
Problem: playing file-like object from gtts
I'm using this code
```
from gtts import gTTS
import pygame
from io import BytesIO

pygame.init()

def say(text):
    tts = gTTS(text=text, lang='en')
    fp = BytesIO()
    tts.write_to_fp(fp)
    fp.seek(0)
    pygame.mixer.init()
    pygame.mixer.music.load(fp)
    pygame.mixer.music.play()
    while pygame.mixer.music.get_busy():
        pygame.time.Clock().tick(10)

text = "This is an example"
say(text)
```
But there is no sound. If I store the gtts output to file.mp3 and do load file.mp3 the output can be heard. But I don't want to write and read "real" files. What can I do. The error message is:
```
Traceback (most recent call last):
  File "/home/jz/Skripte/TTS/googletts-4.py", line 20, in <module>
    say(text)
  File "/home/jz/Skripte/TTS/googletts-4.py", line 14, in say
    pygame.mixer.music.load(fp)
pygame.error: Module format not recognized
```

**Comment by illume** [comment]
I wonder which version of pygame do you have installed?

**Comment by JoergZ2** [comment]
@illume 
pygame==2.0.0

**Comment by illume** [comment]
Thanks for the extra info.

**Comment by JoergZ2** [comment]
In the meantime I found out that pygame can't play mp3 files. On the pygame.org site I found

> Be aware that MP3 support is limited. On some systems an unsupported format can crash the program, e.g. Debian Linux. Consider using OGG instead.
So I tried with python-vlc. VLC can play mp3 but not the file-like object only a stored file. So I generated a small ramdisk and the mp3 file will be stored on the ramdisk and then it can be played. And a ramdisk needs no writings/readings to/from sdcard so I can prevent crashing of sdcard . I hope so....
